utils/usePopper.ts

```diff
@@ -62,7 +77,7 @@ export function usePopper(options) {
     };
   }, [popperRef, referenceRef, modifiers, popperPlacement, positionFixed, eventsEnabled]);
 
-  const style =
+  const style: React.CSSProperties | CSSStyleDeclaration =
```

Comment:
This is a union between `CSSStyleDeclaration` and `React.CSSProperties` because `state.data.styles` is a CSS declaration block that [originates](https://github.com/FezVrasta/popper.js/blob/master/packages/popper/index.d.ts#L107) from the Popper.js.